lib/utils/node_utils.dart

```diff
@@ -40,8 +40,8 @@ class NodeUtils {
 
   static closeEmbeddedNode() async {
     // Release WakeLock
-    if (!Platform.isLinux && await Wakelock.enabled) {
-      Wakelock.disable();
+    if (!Platform.isLinux && await WakelockPlus.enabled) {
```

Comment:
Have you tested WakelockPlus on any OS? 
We may be able to remove the Linux check from these conditional statements.

Reply:
Good point. WakelockPlus [supports Linux](https://pub.dev/packages/wakelock_plus).